qemu/tests/ksm_overcommit.py

```diff
@@ -578,6 +579,9 @@ def split_parallel():
     # As we don't know the number and memory amount of VMs in advance,
     # we need to specify and create them here
     vm_name = params["main_vm"]
+    if arch.ARCH in ('ppc64', 'ppc64le'):
```

Comment:
`arch.ARCH` only checks the host architecture, but you should check the guest architecture here, so use `vm_arch_name` instead.

Reply:
will update it and thanks.